portal-web/test/functional/com/liferay/portalweb/tests/enduseree/formsworkflow/metrics/WorkflowMetrics.testcase

```diff
@@ -608,6 +608,75 @@ definition {
 			stepName = "Update");
 	}
 
+	@description = "Verify the user can set a SLA to Single Approver workflow (LPS-99765)"
+	test SearchPerformanceOfAllAssignees {
+		property portal.acceptance = "false";
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Site Administration",
+			portlet = "Workflow");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		WorkflowMetrics.createUsersAndAssignInstancesWithGroovy(
+			numberOfBlogInstances = "3",
+			numberOfUsers = "3");
+
+		WorkflowMetrics.executeWorkflowScript(scriptName = "CompleteTasksAssignedToUsers.groovy");
+
+		ProductMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Control Panel",
+			portlet = "Metrics");
+
+		WorkflowMetrics.goToWorkflowProcessMetrics(workflowProcessName = "Single Approver");
+
+		Click(locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_TAB");
+
+		Click(
+			key_assigneesNumber = "3",
+			locator1 = "WorkflowMetrics#METRICS_PERFORMANCE_BY_ASSIGNEE_CARD_VIEW_ALL_ASSIGNEES_LINK");
+
+		WorkflowMetrics.searchForAssigneeName(assigneeName = "1user");
+
+		WorkflowMetrics.viewPerformanceByAssignee(
+			assignee = "1user user1",
+			avgCompletionTime = "1min",
+			isNotPresent = "2user user2,3user user3",
+			taskCount = "1");
+
+		WorkflowMetrics.selectWorkloadByAssigneeCardFilterValue(workflowStep = "1Role");
+
+		WorkflowMetrics.viewPerformanceByAssignee(
+			assignee = "1user user1",
+			avgCompletionTime = "1min",
+			isNotPresent = "2user user2,3user user3",
+			taskCount = "1");
+
+		WorkflowMetrics.clearAllFilter();
+
+		WorkflowMetrics.searchForAssigneeName(assigneeName = "1user");
+
+		WorkflowMetrics.selectWorkloadByAssigneeCardFilterValue(workflowStep = "2Role");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "Message#EMPTY_INFO",
+			value1 = "No results were found.");
+
+		WorkflowMetrics.searchForAssigneeName(assigneeName = "something else");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "Message#EMPTY_INFO",
+			value1 = "No results were found.");
+
+		WorkflowMetrics.clearAllFilter();
+
+		AssertElementPresent(locator1 = "//input[@placeholder='Search for Assignee Name']");
```

Comment:
Hi @Adriverforlife 
I believe the locator WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_SEARCH_ASSIGNEE_NAME works in this case. Would you please change it?

Reply:
It would work, the only reason I wrote it this way was because the test wanted me to check if the placeholder was present. If in the future someone rewrites the path for this, the test would no longer be checking for that placeholder. Let me know what you want me to do.

Reply:
I see... as far as I checked the use of METRICS_**WORKLOAD**_BY_ASSIGNEE_SEARCH_ASSIGNEE_NAME, although the word Workload is present, the component used in Performance **is the same**. I don't see a lot of chance of this path to change in the future (unless a designer asks us to).